Source/BottomView/ImageStack.swift

```diff
@@ -3,9 +3,9 @@ import UIKit
 public class ImageStack {
 
   public struct Notifications {
-    public static let imageDidPush = "imageDidPush:"
```

Comment:
Yep, I did few tests to check that it works correctly. Seems that I've just copied selector.